src/components/viewer/Viewer.scss

```diff
@@ -30,60 +31,120 @@ ul.annotation-items {
 li.annotation {
   display: block;
   list-style: none;
-  font-size: $viewer-font-size; /* for most of UI text */
   position: relative;
 }
 
 /* --- Viewer head bar --- */
 .head-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
   padding-top: 4px;
-  clear: both; /* push up controls */
-  font-size: $viewer-font-size;
 }
 
-.comment-priority {
-  float: left;
+.pp-category {
+  @extend %pp-category-font;
+  display: inline;
   margin-left: -($widget-inner-margin + 1px); /* cover the border */
-  padding-top: 3px;
-  padding-bottom: 3px;
+  padding-top: 5px;
+  padding-bottom: 5px;
   padding-left: $widget-inner-margin + 2px; /* indent the type indication */
   padding-right: $widget-inner-margin;
 
-  background-color: rgb(86, 121, 176);
-  font-weight: $font-semi-bold;
-  color: rgb(255, 255, 255);
-  text-transform: uppercase;
-
-  &.priority-normal {
-    background-color: $normal-ref-color;
+  &.category-additional-info {
+    background-color: $additional-info-color;
   }
-  &.priority-warning {
-    background-color: $warning-ref-color;
+  &.category-clarification {
+    background-color: $clarification-color;
   }
-  &.priority-alert {
-    background-color: $alert-ref-color;
+  &.category-error {
+    background-color: $error-color;
   }
 
 }
 
 .comment-date {
-  margin-top: 3px;
+  display: inline-block;
+  min-width: 60px;
   margin-left: 5px;
-  float: left;
+  font-size: $caption-font-size;
+  line-height: 24px;
   font-weight: $font-light;
-  font-size: $viewer-font-size - 2px;
-  color: rgb(86, 121, 176);
-  min-width: 30px;
+  color: $icon-button-color;
+}
+
+.publisher-info {
+  .publisher-demagog {
+    display: flex;
+    max-width: 130px;
+
+    .publisher-name {
+      display: inline-block;
+      max-width: 90px;
+      font-size: $caption-font-size;
+      font-weight: $font-semi-bold;
+      line-height: 1.1em;
+      color: $demagog-color;
+      text-align: right;
+    }
+
+    .publisher-icon {
+      display: inline-block;
+      width: 24px;
+      height: 24px;
+      margin-left: 5px;
+      background-image: url('../../../assets/demagog_logo.png');
+      background-position: center;
+      background-repeat: no-repeat;
+      background-size: cover;
+    }
+  }
 }
 
 .controls {
-  float: right;
+  display: flex;
   padding-right: 5px;
   opacity: 0;
   transition: opacity 0.2s ease-in; // animate on opacity change
 
-  button {
-    @include button-reset()
+  .pp-Button {
```

Comment:
Just to make sure -- there is no need to use uppercase (it will be accessible from code via `ppButtons` anyway).